### PR TITLE
Branch pe d changes

### PR DESCRIPTION
**Pull request by Justinnje**

### CODE REVIEW

File: src/main/java/seedu/address/model/applicant/Applicant.java [hunk 1]
```diff
@@ -147,7 +147,9 @@ public boolean isSameApplicant(Applicant otherApplicant) {
         }
 
         return otherApplicant != null
-                && otherApplicant.getName().equals(getName());
```
Comment (e0543517) [inline src/main/java/seedu/address/model/applicant/Applicant.java:150]: Slight error in java doc.

File: src/main/java/seedu/address/model/applicant/Applicant.java [hunk 2]
```diff
@@ -147,7 +147,9 @@ public boolean isSameApplicant(Applicant otherApplicant) {
...
+                && otherApplicant.getNric().equals(getNric())
```
Comment (e0543517) [inline src/main/java/seedu/address/model/applicant/Applicant.java:150]: Should this be `||` instead of `&&`?

**Review by e0543517 — CHANGES_REQUESTED** [review]
LGTM other than some minor suggestions.

**Review by seanleong339 — APPROVED** [review]
LGTM

**Review by e0543517 — APPROVED** [review]
LGTM

**Pull request merged**